http/handler_store.go

```diff
@@ -342,6 +342,17 @@ func (s *storeHandler) ExecRequest(rw http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (s *storeHandler) GetNodeIdentity(rw http.ResponseWriter, req *http.Request) {
+	store := req.Context().Value(dbContextKey).(client.DB)
+
+	identity, err := store.GetNodeIdentity(req.Context())
+	if err != nil {
+		responseJSON(rw, http.StatusBadRequest, errorResponse{err})
+		return
+	}
+	responseJSON(rw, http.StatusOK, identity)
```

Comment:
Todo: scrub the private key from this response

Reply:
done